projects/kswapfinance/index.js

```diff
@@ -1,75 +1,17 @@
-const sdk = require("@defillama/sdk");
-const abi = require("./abi.json");
-const erc20 = require("../helper/abis/erc20.json");
-const { unwrapUniswapLPs } = require("../helper/unwrapLPs");
-const { transformOkexAddress } = require("../helper/portedTokens");
-
-const liquidityPoolContract = "0xaEBa5C691aF30b7108D9C277d6BB47347387Dc13";
-const depositPoolContract = "0x5E6D7c01824C64C4BC7f2FF42C300871ce6Ff555";
-
-const calcTvl = async (balances, chain, block, pool) => {
-  const poolLength = (
-    await sdk.api.abi.call({
-      abi: abi.getPoolLength,
-      target: pool,
-      chain,
-      block,
-    })
-  ).output;
-
-  const lpPositions = [];
-
-  for (let index = 0; index < poolLength; index++) {
-    const lpOrTokens = (
-      await sdk.api.abi.call({
-        abi: abi.poolInfo,
-        target: pool,
-        params: index,
-        chain,
-        block,
-      })
-    ).output[0];
-
-    const lpOrToken_bal = (
-      await sdk.api.abi.call({
-        abi: erc20.balanceOf,
-        target: lpOrTokens,
-        params: pool,
-        chain,
-        block,
-      })
-    ).output;
-    if (pool == liquidityPoolContract) {
-      lpPositions.push({
-        token: lpOrTokens,
-        balance: lpOrToken_bal,
-      });
-    } else {
-      sdk.util.sumSingleBalance(
-        balances,
-        `okexchain:${lpOrTokens}`,
-        lpOrToken_bal
-      );
-    }
-  }
-  const transformAddress = await transformOkexAddress();
-
-  await unwrapUniswapLPs(balances, lpPositions, block, chain, transformAddress);
-};
-
-const okexTvl = async (timestamp, ethBlock, chainBlocks) => {
-  const balances = {};
-
-  await calcTvl(balances, "okexchain", chainBlocks["okexchain"], liquidityPoolContract);
-
-  await calcTvl(balances, "okexchain", chainBlocks["okexchain"], depositPoolContract);
-
-  return balances;
-};
+const { calculateUsdUniTvl } = require("../helper/getUsdUniTvl.js");
+
+const factory = "0x60DCD4a2406Be12dbe3Bb2AaDa12cFb762A418c1";
+const wokt = "0x8F8526dbfd6E38E3D8307702cA8469Bae6C56C15";
+const whitelist = [
+  "0x382bB369d343125BfB2117af9c149795C6C65C50", // USDT
+  "0x54e4622DC504176b3BB432dCCAf504569699a7fF", // BTCK
+  "0xEF71CA2EE68F45B9Ad6F72fbdb33d707b872315C", // ETHK
+  "0xdF54B6c6195EA4d948D03bfD818D365cf175cFC2", // OKB
+  "0xab0d1578216A545532882e420A8C61Ea07B00B12", // KST
+];
 
 module.exports = {
-  misrepresentedTokens: true,
-  tvl: sdk.util.sumChainTvls([okexTvl]),
+  tvl: calculateUsdUniTvl(factory, "okexchain", wokt, whitelist, "okexchain"),
   methodology:
-    "We count tvl on the mining pool threw LiquidityPool(pairs) contract and farming pool threw DepsitPool(single tokens) contract",
+    "We count tvl on LiquidityPool(pairs) and DepsitPool(single tokens) threw factory contract",
```

Comment:
afaik this is not actually including single sided staking

Reply:
Yes, you are rigth, I forgot to update it!